code/modules/mob/living/carbon/brain/brain.dm

```diff
@@ -95,4 +95,7 @@ I'm using this for Stat to give it a more nifty interface to work with
 			if(istype(src.loc, /obj/mecha))
 				var/obj/mecha/M = src.loc
 				stat("Exosuit Charge:", "[istype(M.cell) ? "[M.cell.charge] / [M.cell.maxcharge]" : "No cell detected"]")
-				stat("Exosuit Integrity", "[!M.health ? "0" : "[(M.health / initial(M.health)) * 100]"]%")
\ No newline at end of file
+				stat("Exosuit Integrity", "[!M.health ? "0" : "[(M.health / initial(M.health)) * 100]"]%")
+
+/mob/living/carbon/brain/can_safely_leave_loc()
+	return FALSE //You're not supposed to be ethereal jaunting, brains
```

Comment:
Juuuuuust make sure you've tested this. I tried using the FALSE define earlier by setting a var to it, however because BYOND, the var would keep being set to 1.